app/models/feature_flag.rb

```diff
@@ -3,7 +3,13 @@ class FeatureFlag
 
   attr_accessor :description, :name, :owner
 
-  PERMANENT_SETTINGS = [].freeze
+  PERMANENT_SETTINGS = [
+    [
+      :service_start,
+      "Allow users to use the service, rather than being sent to the legacy mutual recognition site",
```

Comment:
Can you update the description of the `service_open` flag below, since that also has a similar one?

It should communicate our intent which is to use basic auth pre go-live, and also on the dev/preprod/test environments (most of the time) because those are not meant to be public facing and shouldn't be confused as such.

Also that one should likely be a permanent setting. `:service_start` could be the temporary one I think.

Reply:
Good point, I've amended the commit with that.